spec.emu

```diff
@@ -2962,6 +3356,37 @@ contributors: Ron Buckton, Ecma International
   <emu-clause id="sec-async-function-definitions">
     <h1>Async Function Definitions</h1>
 
+    <h2>Syntax</h2>
```

Comment:
Why is a second definition of AsyncFunctionDeclaration pasted in the middle of the runtime semantics section? There already is a definition of AsyncFunctionDeclaration in the spec and I don't see the existing one being removed in this pull request.